crescent/commands/options.py

```diff
@@ -154,28 +157,36 @@ def option(
 
 @overload
 def option(
-    option_type: USER, description: str = ..., *, name: str | None = ...
+    option_type: USER, description: str = ..., *, name: str | None = ...  # pyright: ignore
 ) -> ClassCommandOption[User]:
     ...
```

Comment:
For these I think it's better to explicitly ignore the parameter with the typevars, so that you don't ignore all the other params. Just put each parameter on its own line, I don't think black will undo it if there's a comment.